newrelic-agent/src/main/java/com/newrelic/agent/tracing/DistributedTracePayloadParser.java

```diff
@@ -53,7 +51,7 @@ public DistributedTracePayloadImpl parse(DistributedTracePayloadImpl outboundPay
             char firstChar = payload.charAt(0);
             if (firstChar != '{') {
                 // This must be base64 encoded, decode it
-                payload = new String(base64.decode(payload), Charsets.UTF_8);
+                payload = new String(Base64.getDecoder().decode(payload), StandardCharsets.UTF_8);
```

Comment:
commons-codec and Java both default to the "standard" encoding (using `+` and `/` instead of the URL-safe `-` and `_`), so it'll be consistent.